src/xc_integrator/local_work_driver/device/local_device_work_driver.hpp

```diff
@@ -89,6 +92,7 @@ class LocalDeviceWorkDriver : public LocalWorkDriver {
 
   void inc_exc( XCDeviceData* );
   void inc_nel( XCDeviceData* );
+  void inc_vxc( XCDeviceData*, density_id );
```

Comment:
Again, either generalize the original function or label this with `uks`. I would opt for the former.